Content.Server/Atmos/Components/DeltaPressureComponent.cs

```diff
@@ -0,0 +1,120 @@
+using Content.Server.Atmos.EntitySystems;
+using Content.Shared.Damage;
+using Content.Shared.FixedPoint;
+
+namespace Content.Server.Atmos.Components;
+
+/// <summary>
+/// Entities that have this component will have damage done to them depending on the local pressure
+/// environment that they reside in.
+/// </summary>
+/// <remarks><para>If the entity does not have an <see cref="AirtightComponent"/>,
+/// simple damage depending on the current pressure will be done instead.</para>
+///
+/// <para>Systems wanting to change these values should go through the <see cref="DeltaPressureSystem"/> API.</para></remarks>
+[RegisterComponent]
+[Access(typeof(DeltaPressureSystem), typeof(AtmosphereSystem))]
+public sealed partial class DeltaPressureComponent : Component
```

Comment:
Given the above, it seems to only make sense to define this component for entities with `AirtightComponent`. As such, maybe we should consider merging this with `AirtightComponent`.

Reply:
I still think that if it doesn't make sense for a `DeltaPressureComponent` to exist without a matching `AirtightComponent`, then they should be one component. It's all about [making illegal states irrepresentable](https://fsharpforfunandprofit.com/posts/designing-with-types-making-illegal-states-unrepresentable/). But I won't make this a blocking issue.